Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
hmm, general guidelines all state you only not track it if you're making gems.  The file does not imply they are already available, the file tells bundler to install on yours exactly what I use to dev against here so we're all on the same page.  Having it makes resolving dependencies quicker for you when you install